trendmicro_deepsecurity/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM komand/python-3-37-plugin:3
```

Comment:
Use the following: komand/python-3-37-slim-plugin

Reply:
done